src/main/java/frc/robot/subsystems/gpm/Arm.java

```diff
@@ -172,9 +179,34 @@ public Arm() {
 
             // put the display on the SmartDashboard
             SmartDashboard.putData("ArmSim", wristDisplay);
-            SmartDashboard.putData(pid);
+            SmartDashboard.putData("arm pid", pid);
         }
 
+		double cachedAngleRad = getAngleRad(); // don't get the angle five times
+		// some checks for the arm position
+		// TODO: decide if we want this
+		if (cachedAngleRad < ArmConstants.MIN_ANGLE_RADS - 0.005 || cachedAngleRad > ArmConstants.MAX_ANGLE_RADS + 0.005) {
```

Comment:
Why does it check if the arm isn't in a narrower range and then if it isn't in a wider one? Also, it disables the arm either way, whether the constant is true or false.

Reply:
The first check is to see whether it's possible at all for the arm to be in this position. If it isn't something is definitely wrong. The second check is to see whether it's at its setpoint. This may indicate something is wrong, but not always, hence the ASSERT_AT_SETPOINT constant.

Reply:
That works. However, can you make the value it has to be over for the first check MIN_ANGLE_RADS - 0.01? The initial setpoint is the min angle, so we should give it the same tolerance.

Reply:
Done.